app/components/marketing/events-section.tsx

```diff
@@ -28,6 +28,14 @@ export function EventsCards({ events }: { events: ExtendedEvent[] }) {
   );
 }
 
+function truncateText(text: string, maxLength: number): string {
```

Comment:
Nice idea to do this! I think this can also be done with pure CSS which then also works without JS, which is always great! However, not a blocker, just a heads-up in case you want to look into that!

Reply:
for one-liner: https://tailwindcss.com/docs/text-overflow

for multiple line: https://tailwindcss.com/docs/line-clamp